.travis.yml

```diff
@@ -12,6 +12,8 @@ jobs:
   include:
     - name: "Test"
       before_script: yarn build
+      env:
+        BABEL_TYPES_8_BREAKING: true
```

Comment:
what does this do?

Reply:
It is a feature flag which enables stricter AST type checking in Babel 8.